modules/pubxaiAnalyticsAdapter.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+Module Name: PubX.io Analytics Adapter
+Module Type: Analytics Adapter
+Maintainer: phaneendra@pubx.ai
+
+# Description
+
+Analytics adapter for prebid provided by Pubx.ai. Contact alex@pubx.ai for information.
+
+# Test Parameters
+
+```
+{
+  provider: 'pubxai',
+    options : {
+      pubxId: 'xxx',
+      hostName: 'https://example.com',
```

Comment:
you probably meant 'example.com', if you leave 'https://example.com' here, it results in a request to `https://https//example.com/analytics/auction?auctionTimestamp=1605274739939&pubxaiAnalyticsVersion=v1.0.0&prebidVersion=v4.15.0-pre` - notice the double protocol.

Reply:
Updated.